archetypes/business-service-springmvc/src/main/resources/archetype-resources/pom.xml

```diff
@@ -26,7 +26,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <java-chassis.version>2.1.2.RELEASE</java-chassis.version>
+    <java-chassis.version>2.5.6</java-chassis.version>
```

Comment:
@liubao68 sorry for my mistake, I may need coffee that time. I will revert this change, and fix java-chassis version in another PR

Reply:
Maybe you need update java chassis version in this PR too. Because you can't change spring boot version without change java chassis version.  Java chassis 2.0.0. may not work with spring boot 2.5.3.  Use Java Chassis 2.6.0 to work with spring boot 2.5.3

Reply:
make sense, I will fixed it